internal/engine/actions/actions.go

```diff
@@ -164,10 +158,7 @@ func shouldRemediate(prevEvalFromDb *db.ListRuleEvaluationsByProfileIdRow, evalE
 
 	// Start evaluation scenarios
 
-	// Case 1 - Do nothing if the evaluation status has not changed and remediation did not errored-out
-	if newEval == prevEval && prevRemediation != db.RemediationStatusTypesError {
-		return engif.ActionCmdDoNothing
-	}
+	// Case 1 - Do not try to be smart about it by doing nothing if the evaluation status has not changed
```

Comment:
FWIW, this will not fix the issue of not updating the PR by force-pushing to it because later on we also have:
```
		if db.RemediationStatusTypesPending == prevRemediation || db.RemediationStatusTypesSuccess == prevRemediation {
			return engif.ActionCmdDoNothing
		}
```
but it should fix the issue you were seeing in staging.

Reply:
Exactly 👍